src/app/modules/inbox/inbox.component.ts

```diff
@@ -109,6 +110,12 @@ export class InboxComponent implements OnInit, OnDestroy {
         });
         this.getIntervieweeList();
         this.getTagFilter();
+        window.addEventListener("storage", broadcast_receive);
+        function broadcast_receive(ev) {
+            if (ev.key == 'changed') {
+                scope.refresh();
```

Comment:
is scope.refresh enough?
does this remove candidate from the current tag? and also update the count of candidate in new tab?

Reply:
rename this function to 'broadcast_receive' == > updateInboxEvent()